calico-cloud/get-started/connect/requirements/system-requirements.mdx

```diff
@@ -17,7 +17,7 @@ Verify that your cluster meets these requirements.
 | --------------------------- | ------------------------------------------------------------ |
 | **Platforms**               | All of the Calico Open Source [Kubernetes deployments](https://docs.projectcalico.org/getting-started/kubernetes/), except: <br/><br/>- OpenShift<br />- IBM Cloud Kubernetes Service (IKS)<br/>- K3s <br />- Most clusters with Windows nodes. (We support Windows nodes on AKS.)<br />- RKE |
 | **Architecture**            | Only AMD64                                                   |
-| **Kubernetes version**      | - Minimum: v1.23<br/>- Maximum: v1.28<br /><br />If you have a later version than the maximum, contact [Support](https://support.tigera.io/). |
+| **Kubernetes version**      | - Minimum: v1.25<br/>- Maximum: v1.29<br /><br />If you have a later version than the maximum, contact [Support](https://support.tigera.io/). |
 | **CNIs**                    | - Calico CNI v3.15 to v3.26<br />- Amazon VPC CNI<br/>- Azure CNI<br/>- GKE CNI |
```

Comment:
As you pointed out I guess we can update the Calico CNI, How about we bump it to v3.20 to v3.27.

Reply:
Done.